### PR TITLE
Pixel perfect Navbar filters

### PR DESCRIPTION
**Pull request by promatik**
I'll give more details with comments on the code 👌

### CODE REVIEW

File: src/resources/assets/css/common.css [hunk 1]
```diff
@@ -472,8 +468,26 @@ form .select2.select2-container {
...
+    border: none !important;
+}
+
+.navbar-filters div.form-group {
+    padding-bottom: 0 !important;
+}
+
+.navbar-filters, .dropdown-menu {
+    --tblr-dropdown-divider-margin-y: .4rem !important;
+}
```
Comment (promatik) [inline src/resources/assets/css/common.css:493]: image

File: src/resources/assets/js/common.js [hunk 2]
```diff
@@ -1,47 +1,12 @@
...
-
-// polyfill for `startsWith` from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
-if (!String.prototype.startsWith) {
-    Object.defineProperty(String.prototype, 'startsWith', {
-        value: function(search, rawPos) {
-            var pos = rawPos > 0 ? rawPos|0 : 0;
-            return this.substring(pos, pos + search.length) === search;
-        }
-    });
-}
```
Comment (promatik) [inline src/resources/assets/js/common.js:12]: https://caniuse.com/?search=startsWith
Only IE doesn't support it.

File: src/resources/assets/js/common.js [hunk 3]
```diff
@@ -1,47 +1,12 @@
...
-
-
-// polyfill for entries and keys from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#polyfill
-if (!Object.keys) Object.keys = function(o) {
-    if (o !== Object(o))
-        throw new TypeError('Object.keys called on a non-object');
-    var k=[],p;
-    for (p in o) if (Object.prototype.hasOwnProperty.call(o,p)) k.push(p);
-    return k;
-}
```
Comment (promatik) [inline src/resources/assets/js/common.js:23]: https://caniuse.com/?search=Object.keys
Even IE supports it 🤷‍♂️

File: src/resources/assets/js/common.js [hunk 4]
```diff
@@ -1,47 +1,12 @@
...
-if (!Object.entries) {
-    Object.entries = function( obj ){
-        var ownProps = Object.keys( obj ),
-            i = ownProps.length,
-            resArray = new Array(i); // preallocate the Array
-        while (i--)
-            resArray[i] = [ownProps[i], obj[ownProps[i]]];
-        return resArray;
-    };
-}
```
Comment (promatik) [inline src/resources/assets/js/common.js:34]: https://caniuse.com/?search=Object.entries
Only IE doesn't support it.

File: src/resources/assets/js/common.js [hunk 5]
```diff
@@ -1,47 +1,12 @@
...
-        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        'X-CSRF-TOKEN': document.querySelector('meta[name="csrf-token"]').content,
     }
 });
 
 // Enable deep link to tab
-document.querySelectorAll('.nav-tabs a').forEach(function (elem) {
-    if(elem.dataset.name === location.hash.substr(1)) (new bootstrap.Tab(elem)).show();
-    elem.addEventListener('click', () => location.hash = elem.dataset.name);
-});
```
Comment (promatik) [inline src/resources/assets/js/common.js:47]: `substr` is deprecated;
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

**Comment by tabacitu** [comment]
Woohoo! OMG this is so much better, thanks @promatik ! Can confirm it's all looking and working nicely:


https://github.com/Laravel-Backpack/CRUD/assets/1032474/ce021e82-0b58-4b7f-96b6-2814130530d9

**Pull request merged**